src/file.cpp

```diff
@@ -1185,7 +1185,6 @@ namespace libtorrent
 		}
 #else
 
-		memset(&m_dirent, 0, sizeof(dirent));
 		m_name[0] = 0;
```

Comment:
I notice you left this line in (`m_name[0] = 0;`) while it was removed in master. is it necessary in `RC_1_1`?

Reply:
Overlooked, sorry. Of course, it is a mistake to assign to a empty `std::string`.

Reply:
Assignment removed.